### ISSUE TITLE
BUG: strip_attr ignores CSS selectors, strip_tags ignores HTML tags

### ISSUE THREAD

**Issue by Kalimbim** [obsidianmd/obsidian-clipper#157]
**Version (please complete the following information):**

- OS: MacOS 15.1
- Browser: Safari
- Web Clipper version: 0.9.5
- Obsidian version: 1.7.6

**Describe the bug**

- `strip_attr:("class")` removes all attributes. 
- `strip_tags:("b")` removes all attributes

**Expected behavior**

-  `strip_attr:("class")` should remove either only `class` or all other attributes (eg `id`).
-  `strip_tags:("b")`  should remove either `<b> … </b>` or all the other tags.

I'm not sure about the intended behavior. [Help](https://help.obsidian.md/web-clipper/filters#HTML+processing) is confusing. Does it strip _only_ or _everything but_ the provided CSS selectors/HTML tags? I'd prefer the former and a different syntax for the latter, but the other way around is also ok.

**URLs where the bug occurs**

I added the examples from Help to a test page: <https://write.as/i52g2kl1kwnghey2.md> 

**To reproduce**

1. Write template with note content:

```
{{selectorHtml:article}}

---

{{selectorHtml:article|strip_attr:("class")}}

---

{{selectorHtml:article|strip_tags:("b")}}
```

![template](https://github.com/user-attachments/assets/7fb9d510-f5c4-45c3-8017-b7255b5c4d4c)

2. Go to: <https://write.as/i52g2kl1kwnghey2.md> 

3. Open Clipper

![result](https://github.com/user-attachments/assets/f6e53187-b168-4fc6-ad40-99d5dc2f5b2e)

I expected to see either this result:

```markdown
<article class="norm h-entry"><h2 id="title" class="p-name">Testing Webclipper</h2><div id="post-body" class="e-content"><div class="test" id="example">Content</div>

<p>Hello <b>world</b>!</p>
</div></article>

---

<article><h2 id="title">Testing Webclipper</h2><div id="post-body"><div id="example">Content</div>

<p>Hello <b>world</b>!</p>
</div></article>

---

<article class="norm h-entry"><h2 id="title" class="p-name">Testing Webclipper</h2><div id="post-body" class="e-content"><div class="test" id="example">Content</div>

<p>Hello world!</p>
</div></article>
```

Or that result:

```markdown
<article class="norm h-entry"><h2 id="title" class="p-name">Testing Webclipper</h2><div id="post-body" class="e-content"><div class="test" id="example">Content</div>

<p>Hello <b>world</b>!</p>
</div></article>

---

<article class="norm h-entry"><h2 class="p-name">Testing Webclipper</h2><div class="e-content"><div class="test">Content</div>

<p>Hello <b>world</b>!</p>
</div></article>

---

Content

Hello <b>world</b>
```

**Your template file**

```json
{
  "schemaVersion": "0.1.0",
  "name": "Testing Testing",
  "behavior": "create",
  "noteContentFormat": "{{selectorHtml:article}}\n\n---\n\n{{selectorHtml:article|strip_attr:(\"class\")}}\n\n---\n\n{{selectorHtml:article|strip_tags:(\"b\")}}",
  "properties": [],
  "triggers": [],
  "noteNameFormat": "{{title}}",
  "path": "Clippings"
}
```

**Comment by Kalimbim** [comment]
Apparently both filters work *without brackets*. Test:

```
{{selectorHtml:article}}

---

{{selectorHtml:article|strip_attr:"class"}}

---

{{selectorHtml:article|strip_tags:"b"}}
```
works almost like expected:

- remove all attributes *except* `class`
- remove all tags *except* `b`

![ex](https://github.com/user-attachments/assets/0cfed118-8f14-438d-9c60-6349fc658152)

Open bugs:
- support notation with brackets for consistency with other filters and for compatibility with help pages
- fix error in help pages for `strip_attr`. Help seems to describe the opposite of the actual behavior

**Comment by kepano** [comment]
Fixed in [0.10.3](https://github.com/obsidianmd/obsidian-clipper/releases/tag/0.10.3)
